torch/lib/THD/base/init_methods/InitMethodTCP.cpp

```diff
@@ -35,8 +35,11 @@ std::string getRandomString()
 
   int fd;
   unsigned int seed;
+  ssize_t bytes_read;
   SYSCHECK(fd = open("/dev/urandom", O_RDONLY));
-  SYSCHECK(read(fd, &seed, sizeof(seed)));
+  SYSCHECK(bytes_read = read(fd, &seed, sizeof(seed)));
```

Comment:
Why do you need to read from `/dev/urandom` (instead of just using the C++11 random library)?